compat/compat_mman.c

```diff
@@ -29,6 +29,9 @@
 #define BT_LOG_TAG "COMPAT-MMAN"
 #include "logging.h"
 
+BT_HIDDEN
+int bt_mman_dummy_symbol;
```

Comment:
Good idea. I'd say ifdef to APPLE and add a comment explaining why that's needed.